### PR TITLE
Make version check not choke in community build.

### PR DESCRIPTION
**Pull request by SethTisue**

### CODE REVIEW

**Comment by sjrd** [comment]
Jenkins, test this please

File: project/Build.scala [hunk 1]
```diff
@@ -161,7 +161,8 @@ object Build extends sbt.Build {
       scalacOptions ++= {
         scalaVersion.value.split('.') match {
           case Array("2", "10", _)                 => Nil
-          case Array("2", "11", x) if x.toInt <= 4 => Nil
+          case Array("2", "11", x)
+            if x.takeWhile(_.isDigit).toInt <= 4   => Nil
```
Comment (sjrd) [inline project/Build.scala]: To follow our coding style, please indent the `if` 4 spaces wrt. the `case`.

**Comment by sjrd** [comment]
While amending the commit, could you also please add Caps to the beginning of sentences (commit title and the paragraph).

Thanks :-)

**Comment by SethTisue** [comment]
fixed, fixed.

**Comment by sjrd** [comment]
LGTM

**Comment by sjrd** [comment]
Jenkins, retest this please

**Pull request merged**